Documentation/network/concepts/routing.rst

```diff
@@ -129,6 +129,12 @@ Requirements on the network
      the routes.  See the guide :ref:`kube-router` on how to achieve this using
      the kube-router project.
 
+  3. If a BGP daemon is running and there is multiple native subnets to 
```

Comment:
hmm, to me this paragraph would make more sense in the `Configuration` section. The preceding points are two alternative approaches, but this is not a third option, but rather additional note to the second one.

Reply:
Thanks, added a new small section describing it as optional since the `Configuration` section states must haves for native routing.